### PR TITLE
added Anisah branch functionality and ready to merge with dev branch

### PR DESCRIPTION
**Pull request by TeresaGobble**
Ready to merge

### CODE REVIEW

File: client/App.js [hunk 1]
```diff
@@ -48,10 +44,10 @@ const App = () => {
       .catch(err => {
         console.log("An error occured while fetching current item", err);
       })
-  }, []);
+  }, [currentProductID]);
```
Comment (clwilshire) [inline client/App.js:47]: Confirm this functions the same as []

File: client/App.js [hunk 2]
```diff
@@ -62,101 +58,126 @@ const App = () => {
       .catch((err) => {
         console.log("An error occured while fetching related products", err)
       })
-  }, []);
+  }, [currentProductID]);
```
Comment (clwilshire) [inline client/App.js:61]: Confirm this functions the same as []

File: client/components/Related-items-and-comparisons/Card.jsx [hunk 3]
```diff
@@ -8,34 +8,90 @@ import Comparison from './Comparison.jsx';
...
-// Product category(req.body.category):
-// Product name (req.body.name):
-// Product price (req.body.default_price):
+  useEffect(() => {
+    let cardObject = {};
+    axios.get(`/products/${props.relatedProductID}`)
+      .then((res) => {
+        cardObject.name = res.data.name;
+        cardObject.default_price = res.data.default_price;
+        return axios.get(`/products/${props.relatedProductID}/styles`) // will not work until Anisah merges changes containing styles axios.get in server
```
Comment (clwilshire) [inline client/components/Related-items-and-comparisons/Card.jsx:20]: Confirm that this does work, now that Anisah's styles axios.get has been merged

**Review by yellowstrings — APPROVED** [review]

**Review by clwilshire — COMMENTED** [review]
Looks good, confirm my comments have been addressed, and maybe add package.json to gitignore for future pull requests?

File: client/components/Related-items-and-comparisons/Card.jsx (hunk 3 above)
Comment (TeresaGobble) [reply client/components/Related-items-and-comparisons/Card.jsx:20]: Good to go!

**Pull request merged**